pontoon/api/serializers.py

```diff
@@ -115,32 +158,44 @@ class Meta(ProjectSerializer.Meta):
     def get_locales(self, obj):
         return [pl.locale.code for pl in obj.project_locale.all()]
 
-    def get_approved_strings(self, obj):
-        return obj.approved
 
-    def get_complete(self, obj):
-        return obj.is_complete
+class NestedIndividualProjectSerializer(ProjectSerializer, TranslationStatsMixin):
+    tags = TagSerializer(many=True, read_only=True)
+    localizations = serializers.SerializerMethodField()
 
-    def get_missing_strings(self, obj):
-        return obj.missing
+    approved_strings = serializers.SerializerMethodField()
+    complete = serializers.SerializerMethodField()
+    missing_strings = serializers.SerializerMethodField()
+    pretranslated_strings = serializers.SerializerMethodField()
+    strings_with_errors = serializers.SerializerMethodField()
+    strings_with_warnings = serializers.SerializerMethodField()
+    total_strings = serializers.SerializerMethodField()
+    unreviewed_strings = serializers.SerializerMethodField()
 
-    def get_pretranslated_strings(self, obj):
-        return obj.pretranslated
+    class Meta(ProjectSerializer.Meta):
+        fields = ProjectSerializer.Meta.fields + ["tags", "localizations"]
 
-    def get_strings_with_errors(self, obj):
-        return obj.errors
+    def get_localizations(self, obj):
+        request = self.context.get("request")
+        if not request:
+            return None
 
-    def get_strings_with_warnings(self, obj):
-        return obj.warnings
+        slug = self.context["view"].kwargs.get("slug")
+        project = get_object_or_404(
+            Project,
+            slug=slug,
+        )
 
-    def get_total_strings(self, obj):
-        return obj.total
+        project_locales = obj.project_locale.stats_data(project=project)
```

Comment:
That way you don't need to re-fetch `project` above:

```suggestion
        project_locales = obj.project_locale.stats_data(project=obj)
```